osgp/protocol-adapter-iec61850/osgp-protocol-adapter-iec61850/src/main/java/org/opensmartgridplatform/adapter/protocol/iec61850/application/services/DeviceManagementService.java

```diff
@@ -108,22 +110,22 @@ public void addEventNotifications(final String deviceIdentification,
 
         LOGGER.info("addEventNotifications called for device {}: {}", deviceIdentification, eventNotifications);
 
-        final RequestMessage requestMessage = new RequestMessage("no-correlationUid", "no-organisation",
+        final RequestMessage requestMessage = new RequestMessage(NO_CORRELATION_UID, NO_ORGANISATION,
                 deviceIdentification, new ArrayList<>(eventNotifications));
 
-        this.osgpRequestMessageSender.send(requestMessage, DeviceFunctionDto.ADD_EVENT_NOTIFICATION.name());
+        this.osgpRequestMessageSender.send(requestMessage, MessageType.ADD_EVENT_NOTIFICATION.name());
     }
 
     /**
      * Get the device output setting (relay configuration) for a given device.
      *
      * @param deviceIdentification
-     *            The device identification.
+     *         The device identification.
      *
      * @return The {@link DeviceOutputSetting} for the device.
      *
      * @throws ProtocolAdapterException
-     *             In case the device can not be found in the database.
+     *         In case the device can not be found in the database.
      */
```

Comment:
I suggest removing this JavaDoc. The only notable piece of information is about the ProtocolAdapterException. Claiming the cause of this exception is risky since future changes can easily invalidate it. The exception documents the underlying cause, this is not supposed to be documented here.